draft-ietf-quic-manageability.md

```diff
@@ -299,38 +301,36 @@ packet for QUIC to be used.
 The content of Initial packets is encrypted using Initial Secrets, which
 are derived from a per-version constant and the client's destination connection
 ID; they are therefore observable by any on-path device that knows the
-per-version constant. They are therefore considered visible in this
-illustration. The content of QUIC Handshake packets are encrypted using keys
-established during the initial handshake exchange, and are therefore not
-visible.
+per-version constant and considered visible in this illustration. The content
+of QUIC Handshake packets are encrypted using keys established during the
+initial handshake exchange, and are therefore not visible.
 
-Initial, Handshake, and the 1-RTT packets transmitted after the handshake
-belong to cryptographic and transport contexts. The Client Completion
-{{fig-init-complete}} and the Server Completion {{fig-hs-complete}} datagrams
-finish these first two contexts, by sending the final acknowledgment and
-finishing the transmission of CRYPTO frames.
+Initial, Handshake, and 1-RTT packets belong to different cryptographic and
+transport contexts. The Client Completion {{fig-init-complete}} and the
+Server Completion {{fig-hs-complete}} flights conclude the Initial and Handshake
+contexts, by sending final acknowledgments and CRYPTO frames.
 
 ~~~~~
 +----------------------------------------------------------+
 | UDP header (source and destination UDP ports)            |
 +----------------------------------------------------------+
 | QUIC long header (type = Initial, Version, DCID, SCID) (Length)
 +----------------------------------------------------------+  |
-| QUIC CRYPTO frame header                                 |  |
+| QUIC Frames (ie: CRYPTO, PADDING, PING)                  |  |
 +----------------------------------------------------------+  |
 | TLS Client Hello (incl. TLS SNI)                         |  |
```

Comment:
does the TLS Client Hello not live within a QUIC frame? this makes it look otherwise in my reading.

Reply:
Thanks, I updated it to be clearer.  I also reverted part of my change and just put "Example" instead of the former "Typical", because the number of ways a client hello can be sequenced is extraordinarily large.